src/mumble/ServerHandler.h

```diff
@@ -34,6 +34,7 @@
 
 #include "Mumble.pb.h"
 #include "MumbleProtocol.h"
+#include "ProtoUtils.h"
```

Comment:
The include should be in the cpp file as it does not seem to be needed in the header.